version/version.go

```diff
@@ -8,7 +8,7 @@ const (
 	// VersionMinor is for functionality in a backwards-compatible manner
 	VersionMinor = 16
 	// VersionPatch is for backwards-compatible bug fixes
-	VersionPatch = 1
+	VersionPatch = 3
```

Comment:
We can’t do this on a side branch; this is unordered WRT existing 5.16.1, not an update to it.

Reply:
@mtrmac Okay dropping version commits